README.md

```diff
@@ -11,10 +11,11 @@ This implements ably broadcaster as a independent service provider library for [
 ## Features
 - Native ably-js support.
 - Low latency for client-events.
-- Disable public channels.
-- Update channel permissions for a user.
+- Update channel permissions for each user.
 - Update token expirty.
-- Fully compatible with [pusher/pusher-based](https://laravel.com/docs/9.x/broadcasting#client-ably) broadcasters, see [migrating section](#migrating-from-old-ablybroadcaster)
+- Disable public channels.
```

Comment:
It's not totally clear from this line what the benefit of disabling public channels is or why someone would need it, can we add some more detail?

Reply:
It's because by default anyone can subscribe to public channels. That means we are giving non-authenticated users permission to listen to the data. Applications with high-security architecture wouldn't want public channels to be available to guest users.